.rubocop.yml

```diff
@@ -4,13 +4,14 @@ AllCops:
     - 'scripts/vendor/**/*'
     - 'platform-tests/src/platform/vendor/**/*'
     - 'tools/metrics/vendor/**/*'
-  TargetRubyVersion: 2.2
-Metrics/BlockLength:
-  # FIXME: Remove when upstream PR is released:
-  # https://github.com/alphagov/govuk-lint/pull/87
-  Enabled: true
-  Exclude:
-    - "**/spec/**/*"
+  TargetRubyVersion: 2.5
 Layout/IndentHeredoc:
-  # FIXME: Remove when we've upgraded to Ruby 2.3
```

Comment:
Now this has been removed we have linting errors which are silenced indefinitely. What are we going to do about these?

Reply:
I don't believe in strongly opinionated style checking, so you might need to ask someone else.